src/main/webapp/resources/js/pages/user/components/UserAccountLayout.jsx

```diff
@@ -1,74 +1,58 @@
 import React from "react";
-import { useDispatch } from "react-redux";
 import { Outlet, useParams } from "react-router-dom";
-import { Col, Row } from "antd";
-import { PageWrapper } from "../../../components/page/PageWrapper";
+import { Col, Layout, PageHeader, Row } from "antd";
 import UserAccountNav from "./UserAccountNav";
 import { setBaseUrl } from "../../../utilities/url-utilities";
 import { useGetUserDetailsQuery } from "../../../apis/users/users";
 import { ContentLoading } from "../../../components/loader";
-import { setUserDetails } from "../services/userReducer";
+import { SPACE_LG } from "../../../styles/spacing";
+import { grey1 } from "../../../styles/colors";
 
 /**
  * React component that layouts the user account page.
  * @returns {*}
  * @constructor
  */
 export default function UserAccountLayout() {
-  const dispatch = useDispatch();
-  const { userId } = useParams();
-  const {
-    data: userDetails,
-    isLoading,
-    isSuccess,
-  } = useGetUserDetailsQuery(userId);
+  const ADMIN_USERS_URL = "admin/users";
 
+  const {Content, Sider} = Layout;
+  const {userId} = useParams();
+  const {data: userDetails = {}} = useGetUserDetailsQuery(userId);
+  const showBack = () => userDetails.admin && document.referrer.includes(ADMIN_USERS_URL, 0);
```

Comment:
Don't make this a function.  Since this really only needs to be a variable.

Reply:
Ooooops... Please see e5e2053ef90284c75d018e3d752f215420a8dd7e.